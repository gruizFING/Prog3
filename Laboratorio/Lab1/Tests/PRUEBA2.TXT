EXPR expr1=(((x+y)*z+10)/((z-1)*x+y))+1+x
EXPR expr2=(x+y)/2-z
EVAL expr1
EVAL expr2
VAR x=2
VAR y=3
VAR z=4
EVAL expr1
EVAL expr2
SAVE prueba2
EXPR expr1=(x+y)*(z+x)
EXPR expr2=x*y+(z+x)/y
EVAL expr1
EVAL expr2
LOAD prueba2
EVAL expr1
EVAL expr2
SALIR
