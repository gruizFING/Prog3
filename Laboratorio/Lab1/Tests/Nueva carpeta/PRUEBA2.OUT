Bienvenido a el mini-sistema:
> OK. expr1 agregada.
> OK. expr2 agregada.
> ERROR: Falta definir variable o expresion.
> ERROR: Falta definir variable o expresion.
> OK. x agregada.
> OK. y agregada.
> OK. z agregada.
> OK. 6.33
> OK. -1.50
> OK. Datos guardados.
> OK. expr1 agregada.
> OK. expr2 agregada.
> OK. 30.00
> OK. 8.00
> OK. Datos cargados.
> OK. 6.33
> OK. -1.50
> 